src/main/scala/com/fasterxml/jackson/module/scala/introspect/ScalaAnnotationIntrospectorModule.scala

```diff
@@ -47,7 +47,7 @@ object ScalaAnnotationIntrospector extends NopAnnotationIntrospector with ValueI
   override def hasIgnoreMarker(m: AnnotatedMember): Boolean = {
     val name = m.getName
     //special cases to prevent shadow fields associated with lazy vals being serialized
-    name == "0bitmap$1" || name.endsWith("$lzy1") || super.hasIgnoreMarker(m)
+    name == "0bitmap$1" || name.endsWith("$lzy1") || name.contains("$default$") || super.hasIgnoreMarker(m)
```

Comment:
I don't know if this is really the best way to filter these methods out, but it's the best way I could find.

Based on the spec at https://docs.scala-lang.org/sips/named-and-default-arguments.html#default-arguments-1

Reply:
thanks - probably the best way to go